scripts/react-tester.js

```diff
@@ -4,30 +4,46 @@ const { execSync } = require( 'child_process' );
 const { mkdirSync, rmdirSync, copyFileSync } = require( 'fs' );
 const { resolve: resolvePath } = require( 'path' );
 const satisfiesSemver = require( 'semver/functions/satisfies' );
-const semverMinor = require('semver/functions/minor')
+const semverMinor = require( 'semver/functions/minor' );
+
+/**
+ *
+ * Usage: `node ./scripts/react-tester.js <command>`
+ *
+ * Commands:
+ *
+ * --browser <name>   Specifies environment to test.
+ *                    Possible values: 'Chrome', 'Firefox', 'BrowserStack_Safari', 'BrowserStack_Edge', 'SSR'. Defaults to: 'Chrome'.
+ * --react <version>  Specifies react version to test. Possible values: 'all', 'current' or specific version. Defaults to: 'current'.
+ * --dry              Runs tests for currently installed version of React without creating temp dir. Use for quick feedback.
```

Comment:
I wonder if `--dry` is not a little misleading name here :thinking: It is usually used for a test run (haha, coincidence? :smile:). But on a serious note, maybe something like `--single` or `--single-run` or sth similar?

And now I wonder, isn't `--dry` kind of duplicating `node ./scripts/react-tester.js` run? Since by default Chrome and current React version is used. So `--dry` only prevents the script from creating tmp dir? How much time we can save thanks to this? :thinking:

Reply:
And it also means that `--dry` ignores `--react` arg which could be also mentioned in the docs above (until we decide to change the way it works)?

Reply:
The idea here is to be able to run tests through `react-tester` but with locally installed `react`. It's much faster and perfect for local development.

Indeed, `dry` is not a perfect choice here although it was supposed to imply that there are no side-effects (no new directory is created, no files are removed, copied, etc.).

Now I'm thinking that this whole `dry` run can be removed in favor of simply using `test:browser` script for local development.